srcpkgs/python-shapely/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'python-shapely'
+pkgname=python-shapely
+version=1.7a1
+revision=1
+wrksrc="Shapely-${version}"
+build_style=python-module
+pycompile_module="shapely"
+hostmakedepends="python-setuptools python3-setuptools python-Cython python3-Cython"
+makedepends="python-devel python-numpy geos-devel"
+depends="python-numpy geos"
+short_desc="Library for manipulation and analysis of geometric objects for Python2"
+maintainer="Karl Nilsson <karl.robert.nilsson@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/Toblerity/Shapely"
+distfiles="https://github.com/Toblerity/Shapely/archive/${version}.tar.gz"
+checksum=d5163f46a99b96c4124b5df2a639dea2dcd7cb45b85128137807e3c9f06d3a16
+
+post_install() {
+	vlicense LICENSE.txt LICENSE
+}
+
+python3-shapely_package() {
+
+	short_desc="${short_desc/Python2/Python3}"
+	pycompile_module="shapely"
+	make_depends="python3-devel python3-numpy geos-devel"
```

Comment:
There is no such thing as make_depends, and makedepends should only be set in the sourcepkg not any subpackage.